llvm-external-projects/iree-dialects/test/Dialect/iree_linalg_ext/convert_to_loops.mlir

```diff
@@ -790,7 +790,7 @@ func.func @NC_to_NCnc(%arg0: memref<128x256xf32>, %arg1: memref<4x8x32x32xf32>)
 // -----
 
 func.func @KC_to_KCck(%arg0: memref<128x256xf32>, %arg1: memref<8x4x32x32xf32>) {
-  iree_linalg_ext.pack %arg0 inner_tiles [32, 32] interchange = [1, 0] into %arg1 : (memref<128x256xf32> memref<8x4x32x32xf32>)
+  iree_linalg_ext.pack %arg0 dims_pos = [0, 1] inner_tiles = [32, 32] interchange = [1, 0] into %arg1 : (memref<128x256xf32> memref<8x4x32x32xf32>)
```

Comment:
If you are going for dims_pos-like, I think you can drop the interchange?

Making a more convoluted examples
```
iree_linalg_ext.pack %arg0 tiled_dims = [2, 0] inner_tiles = [2, 32] into %arg1 : (memref<128x256x2x1000f32> memref<8x256x1x1000x 2x32xf32>)
```

tiled_dims represent the dimensions that get tiled and innert_tiles the final sizes ?

Reply:
yes, correct. `dim_pos` is the index of the dimension to tile while `inner_tiles` is the tiling factor.

Reply:
you want to express `interchange` using `dim_pos`?